docs/concepts/workloads/vm.md

```diff
@@ -130,6 +144,126 @@ The VM must either be powered off or transitioning from powered off to powered o
 
 By default, the VM will be resized once to reflect the new class. That is, if the `VirtualMachineClass` itself is later updated, the VM will not be resized again. The `vmoperator.vmware.com/same-vm-class-resize` annotation can be added to a VM to resize the VM as the class itself changes.
 
+## Encryption
+
+The field `spec.crypto` may be used in conjunction with a VM's storage class and/or virtual trusted platform module (vTPM) to control a VM's encryption level.
+
+### Encrypting a VM
+
+A VM may be encrypted with either:
+
+* the default key provider
+* an `EncryptionClass` resource
+
+#### Default Key Provider
+
+By default, `spec.crypto.useDefaultKeyProvider` is true, meaning that if there is a default key provider and the VM has an encryption storage class and/or vTPM, the VM will be subject to some level of encryption. For example, if there was a default key provider present, it would be used to encrypt the following VM:
+
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: my-vm
+  namespace: my-namespace
+spec:
+  className:    my-vm-class
+  imageName:    vmi-0a0044d7c690bcbea
+  storageClass: my-encrypted-storage-class
+```
+
+#### EncryptionClass
+
+It is also possible to set `spec.crypto.encryptionClassName` to the name of an `EncryptionClass` resource in the same namespace as the VM. This resource specifies the provider and key ID used to encrypt workloads. For example, the following VM would be deployed and encrypted using the `EncryptionClass` named `my-encryption-class`:
+
+```yaml
+apiVersion: vmoperator.vmware.com/v1alpha3
+kind: VirtualMachine
+metadata:
+  name: my-vm
+  namespace: my-namespace
+spec:
+  className:    my-vm-class
+  imageName:    vmi-0a0044d7c690bcbea
+  storageClass: my-encrypted-storage-class
+  crypto:
+    encryptionClassName: my-encryption-class
+```
+
+### Rekeying a VM
+
+Users can rekey VMs by switching to a different `EncryptionClass` or setting a currently non-empty `spec.crypto.encryptionClassName` to an empty value. This will result in the VM being rekeyed using the default key provider if it exists. If it does not, then the VM could be left in a state where it cannot be powered on.
```

Comment:
> This will result in the VM being rekeyed using the default key provider if it exists.

I assume "this" above refers to the latter specifically (i.e. resetting `encryptionClassName` to empty)? Or does "this" also include "switching to a different EncryptionClass" mentioned in the previous sentence that would result using a default key provider as well?